src/makeTxn.ts

```diff
@@ -1138,6 +1141,7 @@ export function makeApplicationCreateTxnFromObject(
         appForeignApps: 'foreignApps';
         appForeignAssets: 'foreignAssets';
         reKeyTo: 'rekeyTo';
+        extraPages: 'extraPages';
```

Comment:
The `RenameProperties` TypeScript generic type renames properties but _passes properties as well_, so this assignment is unnecessary. Adding `'extraPages'` to the `Pick` union is sufficient.

Reply:
gotcha